invenio_cli/commands/packages.py

```diff
@@ -37,6 +37,44 @@ def install_packages(packages):
 
         return steps
 
+    @staticmethod
+    def outdated_packages():
+        """Steps to show outdated packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + ['--outdated']
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Installing python dependencies..."
+            )
+        ]
+
+        return steps
+
+    @staticmethod
+    def update_packages(package):
+        """Steps to update all or one Python packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + [package] if package not None else prefix
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Installing python dependencies..."
```

Comment:
minor: In this case it is installing, but maybe specify that is "updating"?
```suggestion
                message="Updating python dependencies..."
```